parker_harding.yml

```
@@ -0,0 +1,10 @@
+name: "Parker Harding"
+image: "images/parker_harding.jpg"
+title: "Associate IT Professional"
+website: "https://www.linkedin.com/feed/"
+institution: "Job Rotation Program/Center for High Throughput Computing"
+description: "Parker Harding has a dog named Cutie"
+staff: "Rotator/Staff"
```

```suggestion
status: Staff
```

I don't think we use a `description` or `staff` field so let's drop it. We do use the `Status` field, though!